### PR TITLE
Add ssh command for WP-CLI to quickly connect to an environment

### PR DESCRIPTION
**Pull request by schoenkaft**
A command to quickly connect to a server/environment. Defaults to server 1.

```shell
usage: wp ssh <environment> [--server=<server>]
```

Note: most code was taken from [Garp](https://github.com/grrr-amsterdam/garp3/), especially `Grrr\Deploy\Config`. Removed unused code and refactored here and there.

### CODE REVIEW

File: web/app/themes/wordpress-scaffold/lib/Grrr/Deploy/Config.php [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+        return $env_config;
+    }
+
+    protected function _create_path_from_env($env) {
+        return realpath(ABSPATH . '/../../' . self::ENV_CONFIG_PATH . $env . '.rb');
+    }
+
+    protected function _fetch_generic_content() {
+        return file_get_contents(realpath(ABSPATH . '/../../' . self::GENERIC_CONFIG_PATH));
```
Comment (HammenWS) [inline web/app/themes/wordpress-scaffold/lib/Grrr/Deploy/Config.php]: @schoenkaft Vergeet deze niet aan te passen als je de boel naar de root verhuisd.

Comment (HammenWS) [reply web/app/themes/wordpress-scaffold/lib/Grrr/Deploy/Config.php]: En die daar boven ;)

**Review by HammenWS — CHANGES_REQUESTED** [review]
Zoals besproken, verplaatsen naar de root. Voor de rest super nice!

File: web/app/themes/wordpress-scaffold/lib/Grrr/Deploy/Config.php (hunk 1 above)
Comment (schoenkaft) [reply web/app/themes/wordpress-scaffold/lib/Grrr/Deploy/Config.php]: Well... `ABSPATH` always points to the WordPress root directory (`./web/wp/`), so no need to change 😄

**Review by HammenWS — APPROVED** [review]
LGTM!

**Pull request merged**